Packs/QRadar/Integrations/QRadar_v2/QRadar_v2_test.py

```diff
@@ -415,3 +415,41 @@ def test_get_mapping_fields(mocker):
     assert response['Events: Builtin Fields']
     assert response['Assets']
     assert response['Events: Custom Fields']['events']['bloop'] == 'string'
+
+
+class TestGetCustomProperties:
+    error = 'Can\'t send the `filter` argument with `field_name` or `like_name`'
+    client = QRadarClient("", {}, {"identifier": "*", "password": "*"})
+
+    def test_filter_with_field_name(self):
+        from QRadar_v2 import get_custom_properties
+        with pytest.raises(DemistoException, match=self.error):
+            get_custom_properties(self.client, filter='name="hatul"', field_name='b,c')
+
+    def test_filter_with_like_name(self):
+        from QRadar_v2 import get_custom_properties
+        with pytest.raises(DemistoException, match=self.error):
+            get_custom_properties(self.client, filter='name="hatul"', like_name='b,c')
+
+    def test_filter_with_like_name_and_name_field(self):
+        from QRadar_v2 import get_custom_properties
+        with pytest.raises(DemistoException, match=self.error):
+            get_custom_properties(self.client, filter='name="hatul"', field_name='a,g', like_name='b,c')
+
+    def test_filter_only(self, mocker):
```

Comment:
for the positive tests, please also verify the `params` were sent as expected in the request